server/src/main/java/org/elasticsearch/cluster/coordination/PeerFinder.java

```diff
@@ -180,7 +180,7 @@ private void handleWakeUp() {
         }
 
         if (active == false) {
-            logger.trace("ActivePeerFinder#handleWakeUp(): not running");
+            logger.trace("PeerFinder: not running");
```

Comment:
do we even need to say that this is PeerFinder? We have a logger for PeerFinder that already spits out the class name (same thing for other logging occurrences in this class).

Reply:
Of course. I pushed 7948268c09af8bc024a1d29ee2710bd22dec6462.